index.bs

```diff
@@ -711,6 +711,18 @@ Resignation from a Group</h4>
 	See “Exclusion and Resignation from the Working Group” in the W3C Patent Policy [[PATENT-POLICY]]
 	for information about obligations remaining after resignation from certain groups.
 
+<h4 id="emeritus-status">
+Emeritus status</h4>
+        <dfn export lt="Emeritus Status|Emeritus">Emeritus status</dfn> is granted to
+        elected or appointed participants
+        of the <a href="#AB">Advisory Board</a> and <a href="#TAG">Technical Architecture Group</a>
+        and [=Board=] directors
+        once their term has completed.
+        Individuals with [=Emeritus Status=]
+        are authorized to access <a href="#Member-only">Member-only</a> information and
+        <em class="rfc2119">may</em> join the <a href="#ACCommunication">Advisory Committee Mailing List</a> for discussions
+        to participate in discussions.
```

Comment:
```suggestion
        <em class="rfc2119">may</em> join the <a href="#ACCommunication">Advisory Committee Mailing List</a>
        to participate in discussions.
```

Reply:
There are actually two AC mailing lists per process. So this text, as clumsy as it sounds, isn’t actually redundant. /cc @frivoal

Reply:
Ah, I see. How about "may join and participate in the AC mailing list for discussions"?